Source/Core/Core/FreeLookManager.cpp

```diff
@@ -125,6 +168,35 @@ void FreeLookController::LoadDefaults(const ControllerInterface& ciface)
                                       hotkey_string({"Shift", "`Axis Z+`"}));
   m_fov_buttons->SetControlExpression(FieldOfViewButtons::DecreaseY,
                                       hotkey_string({"Shift", "`Axis Z-`"}));
+
+#if defined HAVE_X11 && HAVE_X11
```

Comment:
Was this intended? Seems like you are checki6for HAVE_X11 twice.

Reply:
Update the description, this is unrelated to this review.  Also, this is how it works in `Wiimote` code.

Reply:
Weird, but ok 🤷

Reply:
The two checks are different – one is for `defined HAVE_X11` and one is for `HAVE_X11`. I would assume that checking just `HAVE_X11` would be enough, but if this is how we're doing it in other places in the code then I'd say it's okay to do the same here.